src/_guides/openapi/openapi-format-overlays.md

```diff
@@ -0,0 +1,96 @@
+
+## Creating Overlays with OpenAPI-Format Playground
+
+### Step 1.) Add an OpenAPI document
+
+Copy in some OpenAPI into the box on the left. If you don’t have any handy why not use the [Train Travel API](https://github.com/bump-sh-examples/train-travel-api) by Bump.sh.
+
+![](new-openapi-in/images/guides/openapi-format-overlays/new-openapi-in-playground.png)
+
+### Step 2.) Open the Overlay section
+
+Click the “*OpenAPI Overlay”* button and it will go to a different section. In here you can give your Overlay a name, something that describes what we are trying to do. 
+
+For example: “Add descriptions to tags” is pretty clear about what is going to be achieved by the actions of this overlay. The version number can be whatever. 
+
+![](/images/guides/openapi-format-overlays/overlay-name.png)
+
+### Step 3.) Play with JSONPath
+
+JSONPath is hard and scary at first, but much like RegEx it can be mastered, especially with the help of a handy tool like this. 
+
+To start with JSONPath you use the `$` as the root, then `.` to go to a property. The OpenAPI structure can be navigated like this, so something like `$.info` `$.paths` `$.tags` or `$.components` is a good start.
+
+When you type in a JSONPath in the Target field, the Target Preview will update, so you can be certain you are working on the right bit of the OpenAPI document.
+
+![](/images/guides/openapi-format-overlays/overlay-target.png)
+
+To update a specific tag, JSONPath supports filters, which use a `?(condition)` syntax. Here I am looking through an array of objects for the one which has a name property equal to “Stations”:
+
+![](/images/guides/openapi-format-overlays/overlay-target-better.png)
+
+The syntax worked, I can see the tag object is there in the target preview. Now to work on the action.
+
+### Step 4:) Create the first Action
+
+Overlay actions come in two flavours: “update” or “remove”.
+
+Update is basically a merge, taking the value provided and updating the target to include these values. If it’s an object it will add in the properties provided and override any that were there with the same name.
+
+To add/override the description for a specific tag, we can pass a new `description` property, along with any other properties from the [tags object](/guides/openapi/specification/v3.1/documentation/grouping-operations-with-tags/).
+
+![](OpenAPI%20Format%20A/images/guides/openapi-format-overlays/overlay-update.png)
+
+Remove will not need to take a value, it just removes the target object entirely. For example, removing the tag Payments.
+
+![](OpenAPI%20Format%20A/images/guides/openapi-format-overlays/overlay-remove.png)
+
+There’s a whole lot more than can be done than just mess with tags, but instead of getting stuck into advanced overlaying, let’s see how these two actions run.
+
+### Step 5:) Compare the output
```

Comment:
We can also mention the existence of the "show diff" feature 
![image](https://github.com/user-attachments/assets/b69fdfeb-ce3b-4bea-8494-c97a67d5d848)
And we should highlight that it most likely has some limitations for now, possibly as when applying an overlay any unsupported custom extension is removed, and some reformatting happens. At least that's what I have in my example:
![image](https://github.com/user-attachments/assets/e53ca3d3-eafa-462f-997a-82770e948d80)